src/system/group.cr

```diff
@@ -33,11 +29,7 @@ struct System::Group
   # Returns the group ID for the given group name.
   # Raises `System::Group::NotFoundError` if no group exists.
   def self.gid(group : String) : UInt32
-    group.check_no_null_byte
-    group_struct = LibC.getgrnam(group)
-    return group_struct.value.gr_gid if group_struct
-
-    raise NotFoundError.new("Group with name '#{group.to_s}', was not found.")
+    gid?(group) || raise NotFoundError.new("Group with name '#{group.inspect}', was not found")
```

Comment:
Don't use `inspect` if you already have quotes around: `'"some group"'` feels there's a bit too much quotes.

Also maybe remove the comma as not all other exceptions have that comma, or add the comma to the other ones (I would remove it)

Reply:
ditto

Reply:
My bad. Forgot to remove them.